.circleci/config.yml

```diff
@@ -0,0 +1,137 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.13-stretch-node
+    working_directory: /go/src/github.com/LF-Engineering/insights-connector-jira
+    steps:
+      - checkout
+      - run:
+          name: setup GITHUB token for private repositories
+          command: git config --global url."https://${GITHUB_TOKEN}:x-oauth-basic@github.com/LF-Engineering".insteadOf "https://github.com/LF-Engineering"
```

Comment:
if your go binary doesnt depend upon any private repositories in LFEngineering, you may get rid of this step & the GITHUB_TOKEN variable too.

Reply:
fixed.